utils/oomparser/oomparser.go

```diff
@@ -29,12 +29,11 @@ import (
 	"github.com/google/cadvisor/utils"
 )
 
-var containerRegexp *regexp.Regexp = regexp.MustCompile(
-	`Task in (.*) killed as a result of limit of (.*)`)
-var lastLineRegexp *regexp.Regexp = regexp.MustCompile(
-	`(^[A-Z]{1}[a-z]{2} .*[0-9]{1,2} [0-9]{1,2}:[0-9]{2}:[0-9]{2}) .* Killed process ([0-9]+) \(([0-9A-Za-z_]+)\)`)
-var firstLineRegexp *regexp.Regexp = regexp.MustCompile(
-	`invoked oom-killer:`)
+var (
+	containerRegexp *regexp.Regexp = regexp.MustCompile(`Task in (.*) killed as a result of limit of (.*)`)
+	lastLineRegexp  *regexp.Regexp = regexp.MustCompile(`(^[A-Z]{1}[a-z]{2} .*[0-9]{1,2} [0-9]{1,2}:[0-9]{2}:[0-9]{2}) .* Killed process ([0-9]+) \(([0-9A-Za-z_]+)\)`)
```

Comment:
`{1}` unnecessary

(optional) `[0-9A-Za-z_]` can be replaced with `\w`, although I'm not sure that's more readable.